### DISCUSSION TITLE
Catch error

### DISCUSSION THREAD

**Discussion by clappis** [saul/demofile#245] [Questions] [answered]
if I throw a Error on round_officially_ended event, the class DemoFile catch the exception, but after emit the event "tickent" the app crashs with the exception. if I comment this emit, the next ("end") is emited and i can handle with the error.

This emit "tickend" is util?

**Comment by saul** [comment]
Can you give some example code? I don't fully understand what the problem is

**Comment by saul** [comment, marked as answer]
Duplicate of #96 - has been fixed. Will release tomorrow

**Comment by clappis** [comment]
Thanks a lot
